Dockerfile

```diff
@@ -1,7 +1,8 @@
-ARG GO_IMAGE=goboring/golang:1.14.2b4
-ARG TRIVY_VERSION=0.7.0
+ARG GO_IMAGE=goboring/golang:1.13.15b4
+ARG TRIVY_VERSION=0.11.0
 
 FROM ${GO_IMAGE}
+ARG TRIVY_VERSION
```

Comment:
> An ARG declared before a FROM is outside of a build stage, so it can’t be used in any instruction after a FROM. To use the default value of an ARG declared before the first FROM use an ARG instruction without a value inside of a build stage:

Reply:
If it is not used in a FROM and it is used in a single build stage we should move it to that build stage.